### PR TITLE
create french traduction

### PR DESCRIPTION
**Pull request by superzerg**
I just added the .po and .mo files for the French language.

### CODE REVIEW

**Comment by jrief** [comment]
Merci!

**Comment by corentinbettiol** [comment]
There's some mistakes in the translations, I will add comments with fixes and *may* correct them later.

File: treebeard/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+msgstr "Élément \"%(node)s\" déplacé en temps qu'enfant de \"%(other)s\""
+
+#: treebeard/admin.py:112
+#, python-format
+msgid "Moved node \"%(node)s\" as sibling of \"%(other)s\""
+msgstr "Élément \"%(node)s\" déplacé au même niveau que \"%(other)s\""
+
+#: treebeard/al_tree.py:373 treebeard/mp_tree.py:474 treebeard/ns_tree.py:363
+msgid "Can't move node to a descendant."
+msgstr "L'élément ne peux être déplacé vers un de ces décendant."
```
Comment (corentinbettiol) [inline treebeard/locale/fr/LC_MESSAGES/django.po:37]: un de *ses *descendants

File: treebeard/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+msgid "-- root --"
+msgstr "-- racine --"
+
+#: treebeard/mp_tree.py:382
+msgid ""
+"The new node is too deep in the tree, try increasing the path.max_length "
+"property and UPDATE your database"
+msgstr ""
+"L'élément est trop profond dans l'arbre, essayez d'augmenter la propriété path.max_length "
+"et de mêtre à jour vore base de donnée"
```
Comment (corentinbettiol) [inline treebeard/locale/fr/LC_MESSAGES/django.po:77]: et de *mettre à jour votre base de *données

**Pull request merged**